website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -129,3 +129,26 @@ The SideNav should always be positioned on the left side of the viewport, occupy
 On smaller viewports, the SideNav should collapse to maximize the available real estate on tablet and mobile devices. By tapping the menu icon, users can expand and access the full menu when needed.
 
 ![Responsive side-nav](/assets/components/side-nav/sidenav-position-and-responsive.png)
+
+## Collapse functionality
+
+If the `isCollapsible` property is set to true, a collapse toggle button will be exposed to the end-user allowing them to manually expand and collapse the component. 
+
+![SideNav collapse function](/assets/components/side-nav/sidenav-collapse-interaction.png)
+
+On smaller viewports the SideNav will be rendered in its collapsed state **by default** and will overlay the main page content it its expanded state.
+
+![SideNav overlay on smaller viewports](/assets/components/side-nav/sidenav-overlay-small-viewport.png)
+
+### Collapsed reflow
+
+The collapse functionality of the SideNav gives control to the end-user to unlock more horizontal space in the main page. Thus, the main page content should reflow to occupy this space if the SideNav is in it's collapsed state.
+
+This is handled out of the box by the HDS [AppFrame](/components/app-frame) component, but may need to be accounted for in custom implementations of the component.
```

Comment:
```suggestion
This is handled out of the box by the [AppFrame](/components/app-frame) component, but may need to be accounted for in custom implementations of the application/page layout.
```

Reply:
Also, should the `AppFrame` be added to the `Related` components for this page?

Reply:
Added it, thanks for the suggestion!